packages/components/src/Enumeration/Items/Items.component.js

```diff
@@ -104,11 +104,16 @@ class Items extends React.PureComponent {
 	}
 
 	render() {
+		const calculateListHeight = get(this.props, 'itemsProp.calculateListHeight');
 		const actions = this.props.itemsProp && this.props.itemsProp.actionsDefault;
 		return (
 			<div
+				test-id={'enumeration-items-list'}
```

Comment:
What is this hard coded `test-id` ?

Reply:
It'is a practice to target easily element in TU. It's a comon practice used in TU testing.